frontend/src/index.html

```diff
@@ -143,9 +143,53 @@
     <div class="flex flex-1 mx-auto w-full max-w-screen-xl sm:overflow-y-hidden">
       <div id="filter-panel" class="fixed z-20 top-0 open:flex hidden flex-col w-64 max-w-full h-full text-sm bg-white md:static md:z-auto md:flex md:w-1/5 md:max-w-sm">
         <div class="flex-none p-2 text-center text-teal-800 font-bold bg-submarine-500 uppercase md:hidden">Filter</div>
-        <div class="flex-grow p-2 pb-40 pt-0 overflow-x-hidden overflow-y-auto">
-          <div id="language-list"></div>
+        <div class="flex-none p-2 pb-40 pt-0 overflow-x-hidden overflow-y-auto">
+          <!-- <div id="language-list"></div>
           <div id="sound-quality-list"></div>
+          <div id="duration-menu"></div>
+          <div id="percentage-menu"></div>
+          <div id="category-list"></div>
+          <div id="year-list"></div>
+          <div id="location-list"></div> -->
```

Comment:
Please never keep commented code in the commit. We can always get the previous stage of the code from GIT, therefore no need keeping it commented. Even git diff would be more descriptive with removed lines.

Reply:
Sure, I have removed the commented code.